lib/MPIPreferences/src/MPIPreferences.jl

```diff
@@ -160,23 +180,28 @@ function use_system_binary(;
             If you want to try different name(s) for the MPI library, use
                 MPIPreferences.use_system_binary(; library_names=[...])""")
     end
+
     if isnothing(abi)
         abi = identify_abi(libmpi)
     end
+
     if mpiexec isa Cmd
         mpiexec = collect(mpiexec)
     end
+
     set_preferences!(MPIPreferences,
         "_format" => "1.0",
```

Comment:
I think what we should do is set this to `"1.1"` if any of the new variables are set, but keep it as 1.0 if it is backward compatible (and document it).

Reply:
will also need to update the check at the top of the file.

Reply:
done